lib/gooddata/bricks/middleware/logger_middleware.rb

```diff
@@ -22,10 +22,21 @@
 require_relative 'mask_logger_decorator'
 
 module GoodData
+  class << self
+    attr_accessor :context_manager, :rest_aggregator
+  end
+
   module Bricks
     class LoggerMiddleware < Bricks::Middleware
       def call(params)
         params = params.to_hash
+
+        execution_id = params['GDC_EXECUTION_ID'] || SecureRandom.urlsafe_base64(16)
```

Comment:
warning: Lint/UselessAssignment: Useless assignment to variable - `execution_id`. (rubocop)